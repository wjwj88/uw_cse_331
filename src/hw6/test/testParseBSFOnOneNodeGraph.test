# test parser on one node graph  generic version-Jun Wang 5:00

# build graph
LoadGraph oneNodeGraph oneNodeGraph.tsv

# check graph is built correctly
ListNodes oneNodeGraph
ListChildren oneNodeGraph Amanda
ListChildren oneNodeGraph Jack

# check BFS
FindPath oneNodeGraph Amanda Amanda
FindPath oneNodeGraph Amanda Jack