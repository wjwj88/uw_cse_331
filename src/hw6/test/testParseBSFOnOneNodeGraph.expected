# test parser on one node graph  generic version-Jun Wang 5:00

# build graph
loaded graph oneNodeGraph

# check graph is built correctly
oneNodeGraph contains: Amanda
the children of Amanda in oneNodeGraph are:
the children of Jack in oneNodeGraph are: the node Jack is not in this graph

# check BFS
path from Amanda to Amanda: 
unknown character Jack